### DISCUSSION TITLE
Are you still working on this Project ?

### DISCUSSION THREAD

**Discussion by Cyogenus** [ozankaraali/PiTV#12] [General]
I came across your project and it seems like you have a good understanding of making a STB player. Would you be interested in working on another IPTV player ?

If you get some time please take a look at my open source IPTV player and see what improvements I can make. 

https://github.com/Cyogenus/IPTV-MAC-STALKER-PLAYER

**Comment by ozankaraali** [comment]
I am not interested, good luck with your app. I have job already and I have already made this app. I may only fix issues about my application.
